apps/open-justice/aleph.yml

```diff
@@ -0,0 +1,59 @@
+---
+- hosts:
+    - open-justice
+  become: yes
+
+  roles:
+    - geerlingguy.docker
+
+  vars:
+    postgres_password: "{{ lookup('passwordstore', 'apps/open-justice/{{ env_name }}/POSTGRES')}}"
+    postgres_hostname: "{{ lookup('passwordstore', 'apps/open-justice/{{ env_name }}/POSTGRES subkey=hostname')}}"
+    postgres_username: "{{ lookup('passwordstore', 'apps/open-justice/{{ env_name }}/POSTGRES subkey=username')}}"
+    postgres_database: "{{ lookup('passwordstore', 'apps/open-justice/{{ env_name }}/POSTGRES subkey=database')}}"
+    aleph_secret_key: "{{ lookup('passwordstore', 'apps/open-justice/{{ env_name }}/ALEPH_SECRET_KEY')}}"
+  
+  tasks:
+    - name: apt-get Update
+      apt:
+        update_cache: yes
+
+    - name: Install make
+      apt:
+        name: make
+        state: present
+        
+    - name: Install git
+      apt:
+        name: git
+        state: present
+    
+    - name: Map ElasticSearch Memory
+      become: yes
+      sysctl:
+        name: vm.max_map_count
+        value: 262144
+        state: present
+
+    - name: Clone Aleph repo
+      become: yes
+      git:
+        repo: https://github.com/alephdata/aleph.git
+        dest: /var/open-justice/aleph
+
+    - name: Copy env file
+      ansible.builtin.copy:
+        src: aleph.env
+        dest: /var/open-justice/aleph
+    
+    - name: Add database details to env file
+      ansible.builtin.lineinfile:
+        path: /var/open-justice/aleph/aleph.env
+        regexp: "^ALEPH_DATABASE_URI="
+        line: "ALEPH_DATABASE_URI=postgresql://{{postgres_username}}:{{postgres_password}}@{{postgres_hostname}}/{{postgres_database}}"
+    
+    - name: Add ALEPH_SECRET_KEY
+      ansible.builtin.lineinfile:
+        path: /var/open-justice/aleph/aleph.env
+        regexp: "^ALEPH_SECRET_KEY="
+        line: "ALEPH_SECRET_KEY={{aleph_secret_key}}"
```

Comment:
instead of copy + lineinfile I think you should use https://docs.ansible.com/ansible/latest/collections/ansible/builtin/template_module.html